packages/core-components/api-report.md

```diff
@@ -1447,6 +1454,12 @@ export class UserIdentity implements IdentityApi {
   signOut(): Promise<void>;
 }
 
+// @public
+export const useSidebar: () => SidebarContextType;
```

Comment:
Naming here is a bit broad tbh, wonder if we can make it something around state as well? Thinking `useSidebarOpenState` for symmetry, or something like that.